app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -125,6 +125,7 @@ public void onCreate(Bundle savedInstanceState) {
         toolbar.setNavigationOnClickListener(view -> {
             onSupportNavigateUp();
         });
+        applicationKvStore.putBoolean("first_edit_depict", true);
```

Comment:
Please add a comment explaining what it means when this boolean is true and false.